test-vectors/credentials/verify_failure.json

```diff
@@ -0,0 +1,33 @@
+{
+  "description": "verifiable credential 1.0 verification",
```

Comment:
Can you add a readme with a link to the spec that we're basing this vector off of?